reference/commands/list.rst

```diff
@@ -26,6 +26,11 @@ conan list
                             Remote names. Accepts wildcards
       -c, --cache           Search in the local cache
 
+.. warning::
+
+    In some circumstances, `--format html` might produce semantically different results from `--format json`, like
+    showing a different amount of packages. Use it on your own risk.
```

Comment:
@SSE4 The `--format html` needs to be fixed, but it's not a problem about showing a different amount packages at all, the problem is that it's not showing anything because it's not working.

Reply:
that's exactly that I mean - it may show a different amount, e.g. zero packages, while `--format json` may show more than zero packages. so `not showing anything` falls into that category.